features/util.py

```diff
@@ -133,7 +133,12 @@ def build_debs(
     for filepath in (buildscript, SOURCE_PR_TGZ):
         print("--- Push {} -> {}:/tmp".format(filepath, instance.name))
         instance.push_file(filepath, "/tmp/" + os.path.basename(filepath))
-    instance.execute(["sudo", "bash", "/tmp/" + buildscript])
+    result = instance.execute(["sudo", "bash", "/tmp/" + buildscript])
+    if result.return_code != 0:
```

Comment:
We could use `result.failed` instead

Reply:
Sweet, I didn't realize that existed and didn't think to check - I will update